in-skill-purchase/response/prompts/dummy_response_apla/document.json

```diff
@@ -8,7 +8,7 @@
       "item": {
```

Comment:
Can we change the name of this file to something meaningful than dummy because now we are actively using this as a response for events.